src/helpViewer/helpProvider.ts

```diff
@@ -47,14 +47,14 @@ export class HelpProvider {
 
         // starts the background help server and waits forever to keep the R process running
         const scriptPath = extensionContext.asAbsolutePath('R/help/helpServer.R');
-        // const cmd = `${this.rPath} --silent --slave --no-save --no-restore -f "${scriptPath}"`;
+        const scriptExpression = `"base::source('${scriptPath.replaceAll('\\', '\\\\')}')"`;
```

Comment:
The quoting here does not work on Linux and macOS. The following works, but `replaceAll` might not be useful in the first place.

```typescript
const scriptExpression = `base::source('${scriptPath.replaceAll('\\', '\\\\')}')`;
```